lib/cli.ts

```diff
@@ -143,4 +147,4 @@ const { _: patterns, ...rest } = yargs
   })
   .parseSync();
 
-main(patterns[0] as string, { ...rest });
+main(patterns[0] as string, { ...rest } as Partial<CLIOptions>);
```

Comment:
why is this cast needed? we should avoid casting if at all possible. all the other options have been added without the need for a cast so we should be able to do the same for this option. if `| undefined` is removed from the `coerce` type above does that remove the need for this cast?

Reply:
Hmmm... So, the type that comes out of `yargs` for an option with `array: true` is `(string | number)[] | undefined` ... Our type on CLIOptions is `NameFormatInput[]`.

The way I see this, it doesn't matter if we're typing the input, the truth is that the input will always be `(string | number)[] | undefined` ... Indeed, the choices are passed to it, but the input can be anything, which needs to be validated via a type-guard... I'll address this 😉

Reply:
Addressed: [e67879f](https://github.com/skovy/typed-scss-modules/pull/164/commits/e67879fbfbadddc3b495be59a6183d29aa90b7bd)